pkg/sdk/testint/basic_object_tracking_integration_test.go

```diff
@@ -0,0 +1,94 @@
+package testint
+
+import (
+	"context"
+	"fmt"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/snowflakedb/gosnowflake"
+	"github.com/stretchr/testify/require"
+	"strings"
+	"testing"
+)
+
+// Research for basic object tracking done as part of SNOW-1737787
+
+// https://docs.snowflake.com/en/sql-reference/parameters#query-tag
+func TestInt_ContextQueryTags(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	sessionId, err := client.ContextFunctions.CurrentSession(ctx)
+	require.NoError(t, err)
+
+	queryTag := "some query tag"
+	require.NoError(t, client.Parameters.SetSessionParameterOnAccount(ctx, sdk.SessionParameterQueryTag, queryTag))
```

Comment:
this does not invoke https://docs.snowflake.com/en/sql-reference/sql/alter-session but alter account and in cleanup we clean a session. Let's use ALTER SESSION SET instead of SetSessionParameterOnAccount.

Reply:
that's the reason why we currently have these fails on the CI:
```
users_acceptance_test.go:54: Step 1/2 error: Check failed: check 3/35 error:
        data.snowflake_users.test parameters assertion [33/59]: failed with error: Attribute 'users.0.parameters.0.query_tag.0.value' expected "", got "some query tag"
 ```

Reply:
Changed to session param change instead of account one.